07LessonSeven/homework/css/style.css

```diff
@@ -1,5 +1,6 @@
 body {
   font-family: sans-serif;
+  width:100%;
```

Comment:
This made a difference on mine.. This changes the font of the page content as well as the width of the container.  

    On Saturday, August 27, 2016 9:46 PM, Erik Zambrano <notifications@github.com> wrote:
 

 In 07LessonSeven/homework/css/style.css:> @@ -1,5 +1,6 @@
>  body {
>    font-family: sans-serif;
> +  width:100%;
I unchecked this property using developer tools I don't see any difference—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.